docs/design2.md

```diff
@@ -0,0 +1,102 @@
+# Key Transparency 2.0
+
+## Objective
+
+Key Transparency currently sufferes from a design-space tradeoff in two dimentions between
+speed and verification bandwidth.  Each snapshot of the key-value merkle tree creates an additional
+point in time that clients must audit when inspecting their account history. 
+Using Key Transparency in this mode requires the presence of powerful auditors who can verify
+that no information has been lost between all snapshots.
+
+| Update Speed | Trust, then Verifiy | Verify First |
+|--------------|---------------------|--------------|
+| hr           | Tractable Bandwidth | Unusable     |
+| min          |                     | Intractable  |
+| s            |                     | Intractable  |
+
+
+Key Transparency 2.0 is a new set of algorithms that resolves this fundamental tradeoff by sharing 
+the task of auditing between pairs of clients. As a result, all configurations result in efficiently
+auditable data structures, without the presence of 3rd parties. 
+
+We expect that many applications will want to prioritize speed and reliability by using a trust, then verify mode, but because the client operations are very similar, users could have the option to wait a bit longer in order to have high confidence that they are using data from a snapshot that is widely available. 
```

Comment:
Can you clarify similar to what/or is it client operations when prioritising speed over latency and vice-versa, or something else?